Source/DafnyCore/Resolver/PreType/PreTypeResolve.cs

```diff
@@ -237,6 +294,25 @@ public enum Type2PreTypeOption { GoodForInference, GoodForPrinting, GoodForBoth
       return DPreType.IsReferenceTypeDecl(dp.Decl);
     }
 
+    /// <summary>
+    /// Add to "ancestors" every TopLevelDecl that is a reflexive, transitive parent of "d",
+    /// but not exploring past any TopLevelDecl that is already in "ancestors".
+    /// An ancestor
+    void ComputeAncestors(TopLevelDecl d, ISet<TopLevelDecl> ancestors) {
+      if (!ancestors.Contains(d)) {
+        ancestors.Add(d);
+        if (d is TopLevelDeclWithMembers dm) {
```

Comment:
nit: variable names `d`, `decl`, `dm` are used interchangeably for `TopLevelDecl` and `TopLevelDeclWithMembers`.

Reply:
Thanks. I changed these to use (our now more typical) longer names.